cscs-checks/cuda/multi_gpu.py

```diff
@@ -35,25 +35,26 @@ def __init__(self, **kwargs):
                     sn.extractsingle(self._xfer_pattern(xfer_kind, device),
                                      self.stdout, 2, float, 0)
 
-        kesch_cn = {}
-        daint_gpu = {}
-        dom_gpu = {}
+        self.reference = {}
         for device in range(self.num_gpus_per_node):
-            kesch_cn['perf_h2d_%i' % device] = (7213, -0.1, None)
-            kesch_cn['perf_d2h_%i' % device] = (7213, -0.1, None)
-            kesch_cn['perf_d2d_%i' % device] = (137347, -0.1, None)
-            dom_gpu['perf_h2d_%i' % device] = (11648, -0.1, None)
-            dom_gpu['perf_d2h_%i' % device] = (12423, -0.1, None)
-            dom_gpu['perf_d2d_%i' % device] = (373803, -0.1, None)
-            daint_gpu['perf_h2d_%i' % device] = (11648, -0.1, None)
-            daint_gpu['perf_d2h_%i' % device] = (12423, -0.1, None)
-            daint_gpu['perf_d2d_%i' % device] = (305000, -0.1, None)
-
-        self.reference = {
-            'kesch:cn': kesch_cn,
-            'daint:gpu': daint_gpu,
-            'dom:gpu': dom_gpu
-        }
+            self.reference['kesch:cn:perf_h2d_%i' % device] = \
```

Comment:
Rename `device` to `dev` so that everything fits in single line.

Reply:
The last line in the loop has a large number and takes exactly 80 rows with `dev`, which is one beyond the maximum. What is it better, to cut that line with a '\' or to change `dev` for `d` on the loop?

Reply:
Yes, rename it to `d`.